compiler/rustc_mir_build/src/thir/pattern/const_to_pat.rs

```diff
@@ -476,7 +465,7 @@ impl<'tcx> ConstToPat<'tcx> {
                         PointerPattern
                     );
                 }
-                PatKind::Constant { value: cv }
+                return Err(FallbackToConstRef);
```

Comment:
This one is already doing random things at runtime depending on opts. We are also already not using it for exhaustiveness.